plugins/inputs/kafka_consumer/sample.conf

```diff
@@ -13,14 +23,14 @@
   ## When set this tag will be added to all metrics with the topic as the value.
   # topic_tag = ""
 
+  ## The list of Kafka message headers that should be pass as metric tags
+  ## works only for Kafka version 0.11+, on lower versions the message headers 
+  ## are not available
+  # msg_headers_to_tags = ['Country','City']
```

Comment:
```suggestion
  # msg_headers_to_tags = []
```

Reply:
I thought there should be as example. Thanks